### ISSUE TITLE
[Accessibility][Usability]A11y_VSCode_UserSnippets_ScreenReader: VoiceOver does not announce anything when we enter any value in the edit field of the combobox. 

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52384]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : HighSierra10.13.5 

**Additional Details:**
ScreenReader: VoiceOver

**Repro Steps:**
1. Launch VS Code while VoiceOver is ON.
2.Navigate to Settings icon in Activity Bar->Press Enter->Select "User Snippets" from dropdown..

**Actual:**
 VoiceOver does not announce anything when we enter any value in the edit field of the combobox and also how many data found in the treeview after filtering.

**Expected:**
VoiceOver should read first whatever user types in the edit field and also how many data found in the treeview matching those words whatever user has typed.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not get the information about whether any data been found matching those words or not.

Attachment for Reference: 
[A11y VSCode UserSnippets VoiceOver not reading keys.pptx](https://github.com/Microsoft/vscode/files/2117978/A11y.VSCode.UserSnippets.VoiceOver.not.reading.keys.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52383
